huxley/www/js/components/RegistrationView.js

```diff
@@ -724,4 +614,484 @@ const RegistrationPhoneInput = React.createClass({
   },
 });
 
-module.exports = RegistrationView;
+const RegistrationHeader = React.createClass({
```

Comment:
Always put components into their own files; the code becomes impossible to read/maintain if there are a dozen components in one file